### PR TITLE
[backport-i18n] Backport i18n from MetaDeploy.

### PR DESCRIPTION
**Pull request by jgerigmeyer**
Based on: https://github.com/SFDO-Tooling/metadeploy/compare/9e439157d26a8cbd6040066117c21e6700a4527d...e41ac55

ocelot

### CODE REVIEW

**Comment by jgerigmeyer** [comment]
@wlonk I think this is good to go, except missing pytest coverage on three files:

- .../admin_middleware.py
- .../adminapi/api.py
- .../adminapi/urls.py

**Comment by davisagli** [comment]
admin_middleware.py and the serializer and viewset classes for the admin API probably belong in sfdo-template-helpers rather than here. They are components that we would like to reuse between projects and test once rather than duplicating code. The _configuration_ of them (i.e. settings and urlconf) can go here in sfdo-template though. I haven't reviewed this yet but your last comment caught my attention.

**Comment by jgerigmeyer** [comment]
@davisagli Sounds good; I sort of assumed as much. My thinking in copying them over here was that either 1) it would inspire @wlonk or you to abstract them to sfdo-template-helpers, or 2) it would hang around here and when we port to sfdo-template-helpers we'd make the change here too. I'm okay just skipping code coverage on them for now, or using this as an excuse to write more abstracted tests for them.

File: {{cookiecutter.project_slug}}/config/settings/base.py [hunk 1]
```diff
@@ -10,19 +10,29 @@
...
+from typing import List
 
 import dj_database_url
 from django.core.exceptions import ImproperlyConfigured
 
 BOOLS = ("True", "true", "T", "t", "1", 1)
 
 
-def boolish(val):
+def boolish(val: str) -> bool:
```
Comment (wlonk) [inline {{cookiecutter.project_slug}}/config/settings/base.py:24]: Not strictly true. `val: Any` is more accurate.

**Review by wlonk — APPROVED** [review]

**Comment by wlonk** [comment]
"Inspire" me 😆 

I'm inclined to leave them in here until we make the changes to this to actually use template-helpers. Which is something we should get in the backlog to do, yeah?

**Pull request merged**